dev.env.default

```diff
@@ -1,6 +1,6 @@
 # Environment variables needed for running this project
 # copy this file to .env in this directory for it to be picked up by docker-compose
-POSTGRES_DB=tapp_development
-POSTGRES_USER=tapp
+POSTGRES_DB=cp_development
```

Comment:
tapp and cp are separate postgress dbs.. do you think it would be okay to leave the users the same?